robottelo/ui/browser.py

```diff
@@ -1,6 +1,15 @@
+"""Tools to help getting a browser instance to run UI tests."""
+import time
+
 from robottelo.config import settings
 from selenium import webdriver
 
+try:
+    import docker
+except ImportError:
+    # Let fail later if not installed
```

Comment:
I was not able to think on a better approach, yesterday was a long day... Will try to come up with something better today.